### DISCUSSION TITLE
Score update animation still appearing in bottom-left when playing CTB mode.

### DISCUSSION THREAD

**Discussion by eggooooo** [ppy/osu#16249] [Q&A] [answered]
The score update animation that's usually in the bottom-left of the screen still seems to appear when playing CTB even though the combo counter is always following the catcher and doesn't appear like this in stable (mentioning stable because this is a classic styled skin) and not in the bottom-left. 

https://user-images.githubusercontent.com/64386789/147398387-27462a64-e3cb-4331-9125-c5481eec4c64.mp4

**Comment by bdach** [comment, marked as answer]
#16250
